bot/resources/tags/system-python.md

```diff
@@ -0,0 +1,18 @@
+---
+embed:
+    title: "System Python"
+---
+
+*Why Avoid System Python for Development on Unix-like Systems:*
+
+- **Critical System Operations Dependency:** Altering the system interpreter may harm system functioning irreversibly.
```

Comment:
Are we altering the interpreter at all? Breaking interactions with system Python tend to create dependency conflicts, which is the principle reason that we tend to dissuade individuals from utilizing the System Python installation.

Reply:
They may attempt to alter it by 'updating' it or deleting files and libraries they find unnecessary, both of which have already happened in PyDis. 
Although the chances of it happening are significantly smaller than a simple dependency break, I don't see the harm in highlighting those risks as a stronger way to dissuade them from even trying.

Reply:
```suggestion
- **Critical Operation System Dependencies:** Altering the system Python installation may harm internal operating system dependencies. 
```